src/components/unique-token/UniqueTokenAttributes.tsx

```diff
@@ -1,64 +1,78 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { sortList } from '../../helpers/sortList';
 import { magicMemo } from '../../utils';
-import Tag from '../Tag';
+import Tag from './Tag';
 import { Inline } from '@rainbow-me/design-system';
 import { UniqueAsset } from '@rainbow-me/entities';
-
-interface AttributeItemProps {
-  color: string;
-  disableMenu?: boolean;
-  trait_type: string;
-  slug: string;
-  value: string | number;
-  max_value?: string | number;
-}
-
-const renderAttributeItem = ({
-  color,
-  disableMenu,
-  trait_type: type,
-  slug,
-  value,
-  max_value: maxValue,
-}: AttributeItemProps) =>
-  type && value ? (
-    <Tag
-      color={color}
-      disableMenu={disableMenu}
-      key={`${type}${value}`}
-      maxValue={maxValue}
-      slug={slug}
-      text={value}
-      title={type}
-    />
-  ) : null;
+import isHttpUrl from '@rainbow-me/helpers/isHttpUrl';
+import transformUniqueAssetTraitsForPresentation from '@rainbow-me/helpers/transformUniqueAssetTraitsForPresentation';
+import uniqueAssetTraitDisplayTypeCompareFunction from '@rainbow-me/helpers/uniqueAssetTraitDisplayTypeCompareFunction';
 
 interface UniqueTokenAttributesProps {
   color: string;
-  disableMenu?: boolean;
+  isPoap?: boolean;
```

Comment:
it looks like were still only using this for disabling the menu, if thats the case I think it makes sense to keep the prop as it was and decide this at use, gives up more flexibility if we need to disable for another use case